ClemBot.Bot/bot/api/message_route.py

```diff
@@ -1,5 +1,7 @@
 import typing as t
 
+from discord import user
```

Comment:
Why do we need this?

Reply:
I think it was added by VSCode automatically sometime during my trials of how to do this lol

Reply:
deleted the line and tested. works fine